assets/js/atomic/blocks/product-elements/product-details/block.json

```diff
@@ -6,8 +6,9 @@
 	"description": "A block that allows your customers to see details and reviews about the product.",
 	"category": "woocommerce",
 	"keywords": [ "WooCommerce" ],
-	"supports": {},
-	"attributes": {},
```

Comment:
How come we're removing the `attributes` property here and adding it in `product-image-gallery/block.json` below?

I couldn't find the specific commit that made this change so I'm just curious! 🙂

Reply:
I updated this PR multiple times these weeks, so I'm unsure who and when the attributes key was removed.

I agree with you, we should remove the empty attributes on the product image gallery block.json I created a PR https://github.com/woocommerce/woocommerce-blocks/pull/8825